node/subsystem-types/src/messages.rs

```diff
@@ -910,3 +910,15 @@ pub enum GossipSupportMessage {
 	#[from]
 	NetworkBridgeUpdateV1(NetworkBridgeEvent<protocol_v1::GossipSuppportNetworkMessage>),
 }
+
+/// PVF checker message.
+///
+/// Currently non-instantiable.
+#[derive(Debug)]
+pub enum PvfCheckerMessage {}
+
+impl BoundToRelayParent for PvfCheckerMessage {
+	fn relay_parent(&self) -> Hash {
+		match *self {}
+	}
+}
```

Comment:
I don't quite understand the purpose of this yet

Reply:
Me neither! I just repeat the picture I saw with the other similar item: `BitfieldSigningMessage`.

Reply:
You don't need to implement the trait if you don't use the `JobTrait`.

Reply:
well, let's remove it then

Reply:
ready https://github.com/paritytech/polkadot/compare/84a142f7a0373cdf4c82b6afa6f5c381862b35ce..fc7e458856930bc5b5baf41fb7f8809b9f1a00e0